### PR TITLE
Make note of important Marathon breaking changes (DCOS_OSS-4243)

### PR DESCRIPTION
**Pull request by timcharper**
Updates the changelog.

DCOS_OSS-4243

### CODE REVIEW

**Comment by timcharper** [comment]
@mesosphere-mergebot label Ready for Review

File: CHANGES.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ Format of the entries must be.
 
 * Replaced dcos-metrics with Telegraf (DCOS_OSS-3714)
 
+* Requests to Marathon's events API (/v2/events when queried directly) for standby instances no longer proxy, but instead respond with a redirect. Clients consuming Marathon's events API should be updated to follow redirects. Component such as updated to versions that follow redirects; for example, Marathon-LB should be updated to at least version v1.12.3. [See more details](https://github.com/mesosphere/marathon/blob/master/changelog.md#non-leaderstandby-marathon-instances-respond-to-v2events-with-a-redirect-rather-than-proxy).
```
Comment (orsenthil) [inline CHANGES.md]: Component must be updated to

Comment (orsenthil) [inline CHANGES.md]: Please mention the JIRA ID at the end of the CHANGELOG entry.

File: CHANGES.md [hunk 2]
```diff
@@ -42,6 +42,10 @@ Format of the entries must be.
...
+
+* Marathon will decline offers for agents in maintenance mode now, by default. [See more details](https://github.com/mesosphere/marathon/blob/master/changelog.md#maintenance-mode-support-production-ready-now-default).
```
Comment (orsenthil) [inline CHANGES.md]: Please add JIRA ID for this changelog entry.

File: CHANGES.md (hunk 1 above)
Comment (timcharper) [reply CHANGES.md]: Component was already updated. This is describing blocking changes that should have been advertised.

Comment (timcharper) [reply CHANGES.md]: Which JIRA? Marathon's?

File: CHANGES.md (hunk 2 above)
Comment (timcharper) [reply CHANGES.md]: ditto ^

**Comment by orsenthil** [comment]
@mesosphere-mergebot label work in progress

**Comment by timcharper** [comment]
This was advertised via another channel. Closing.

**Pull request closed without merging**